fastai/medical/imaging.py

```diff
@@ -381,7 +381,7 @@ def as_dict(self:DcmDataset, px_summ=True, window=dicom_windows.brain):
     return res
 
 # Cell
-def _dcm2dict(fn, **kwargs): return fn.dcmread().as_dict(**kwargs)
+def _dcm2dict(fn,window,**kwargs): return fn.dcmread().as_dict(window=window, **kwargs)
```

Comment:
should `window` have a default argument?

Reply:
I used `window` because `as_dict` uses window so thought to keep it consistent.

Reply:
So you'd need to have the same thing here, right? Otherwise `_dcm2dict` can't be called without a `window` param

Reply:
To be honest that is what I thought but this works.  Out of the box calling:

`dicom_dataframe = pd.DataFrame.from_dicoms(items)`  - works.

`dicom_dataframe = pd.DataFrame.from_dicoms(items, window=dicom_windows.mediastinum)` - raises an error that window is not defined.

Updating to:


`@patch
def _dcm2dict(fn, window, **kwargs): return fn.dcmread().as_dict(window=window, **kwargs)`

Now this seems to work:
`dicom_dataframe = pd.DataFrame.from_dicoms(items, window=dicom_windows.mediastinum)`

Ill look more into it but I made sure that the `img_pct_window` values were updating depending on the window.

![dict](https://user-images.githubusercontent.com/25020209/104033219-197baa80-5184-11eb-9a4d-76d6276a4dde.PNG)



![dict2](https://user-images.githubusercontent.com/25020209/104033239-213b4f00-5184-11eb-9787-48355a718aa6.PNG)

Reply:
Is there a reason you don't want to add the default param to this function? It would save some time for both of us, unless there's a reason you think it's a bad idea. I'd love get this PR merged

Reply:
Just trying to minimize the code :), will fix asap thank you